api/v1alpha1/controllermanagerconfig_types.go

```diff
@@ -125,6 +125,11 @@ type ControllerManagerConfigurationSpec struct {
 	// If specified, uses a different parent id template for linking pods to nodes
 	// +optional
 	ParentIDTemplate string `json:"parentIDTemplate,omitempty"`
+
+	// If specified, only syncs the specified CR types. Defaults to all.
+	// Options, one or more of: clusterspiffeids, clusterfederatedtrustdomains, clusterstaticentries
+	// +optional
+	SyncTypes []string `json:"syncTypes,omitempty"`
```

Comment:
What would you think about this being another struct with three named booleans instead of a list of strings? It would be more self documenting and probably more strongly validated. The struct would also be easier to consume than the strings.Contains calls, which are a little fragile and subject to typos.

Reply:
Sounds good to me.

Reply:
Done